### PR TITLE
Expanded setup to allow unofficial EL clients

### PR DESCRIPTION
**Pull request by Nabsku**
This allows users to use unofficial EL clients in Lighthouse (for example the public Besu package).

The user has to actively toggle the advanced mode to get a free-text field. Basic is pre-selected.

I've also changed the order of items in the setup-wizard to make it less confusing.

### CODE REVIEW

File: setup-wizard.yml
Comment (alexpeterson91) [inline setup-wizard.yml]:
```suggestion
      Set this to **"advanced"** to pass in an arbitrary Execution Client Engine-API or leave on **basic** to choose one below.
```

File: setup-wizard.yml
Comment (alexpeterson91) [inline setup-wizard.yml]:
```suggestion
      Advanced and Development Use Only: Please only enable this option if you are confident you know what you are doing, if you have any doubts, please use the basic setting.
```

File: setup-wizard.yml
Comment (alexpeterson91) [inline setup-wizard.yml]:
```suggestion
      You can find the Engine API in the Execution Client's Package Info.
```

**Review by alexpeterson91 — CHANGES_REQUESTED** [review]
Check over my review if you think its acceptable, do not merge yet awaiting final approval.

**Comment by alexpeterson91** [comment]
dont wanna mention besu yet because i dont want people jumping to install it and think its all 100% good to users are gonna need to rename their volume after deleting the public package and then ssh in and rename the volume to use the DNP and not have to resync, i did that personally not hard but not easy most people dont have root access even if they know how to access the node and basic linux CLI. if theyre using besu or something else they know they need to do something and will find this very helpful.

**Comment by Nabsku** [comment]
ACK and LGTM!

File: setup-wizard.yml
Comment (alexpeterson91) [inline setup-wizard.yml]:
```suggestion
  - id: config_mode
```
despite it not mattering in a yaml Im gonna just keep the style consistent making the id the first entry for each option, just flipping the title and id here before i push to the other clients

File: setup-wizard.yml
Comment (alexpeterson91) [inline setup-wizard.yml]:
```suggestion
    title: Execution Client Selection Configuration Mode
```
again keeping with the preexisting style

**Review by alexpeterson91 — CHANGES_REQUESTED** [review]
apply my comments to enforce the same code ordering style in the setup wizards

**Review by alexpeterson91 — APPROVED** [review]

**Pull request merged**
